common/src/main/java/org/wso2/carbon/testgrid/common/constants/TestGridConstants.java

```diff
@@ -13,4 +13,6 @@
     public static final String TEST_TYPE_TESTNG = "TESTNG";
 
     public static final String JMTER_SUFFIX = ".jmx";
+
```

Comment:
Remove the additional blank line here

Reply:
Fixed with 69b668b93159143b0eb766bf2210e1ad31f5f5e7